src/Connector.js

```diff
@@ -71,23 +73,20 @@ class Connector {
   async setupThingConnection(id) {
     const gatewayClient = await this.createConnection();
     try {
-      const thingClient = await this.resetTokenAndConnect(gatewayClient, id);
+      const thingClient = await this.createConnection();
```

Comment:
I think you got confused because in the previous version we would need to create an individual connection per thing in order to do some operations, e.g `updateSchema` (`thing` updating itself) and `publishData` (`thing` publishing its data). In this new version, we only need to create one connection so that we can operate by using the `user` token.

Reply:
Done :smile: